### PR TITLE
React Metrics webpage - Full site

### PR DESCRIPTION
**Pull request by angeluray**
# Hi good reviewer, happy to see you here! 💯 

Sweet greeting

## Basic check ✔️ 

- Created REACT app.
- Created a new branch according to the GitHub Flow rules.
- Added a .gitignore file to make sure any irrelevant files are ignored by git.
- Added redux workspace including toolkit plus all the required tools.
- Created routes and view components (Homepage, Item Details). Used <NavLink /> for the page navigation links, and updated routes according to the selected item(Cryptocurrency), when the user clicks (or taps) on a category item, the application navigates to the details page.
- Retrieved data from the API to show detailed data in the category (Details section).
- Fetched data from the CoinGeckco API endpoints for displaying the list on the Homepage and displaying the specific item in the details section.
- Added "Back" button (<) functionality, if the user clicks on the button it navigates to the home page.
- Added actions and reducers.
- Added Unit Tests for main components.
- Added styles to both sections.

## Note ⚠️ : 

Here's the link to the full site deployed: https://tourmaline-kitsune-d693cd.netlify.app/
However, you dear reviewer can find it in the README file as well. Thank you for taking the time and review my project.

### CODE REVIEW

File: src/components/cryptohome.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchCryptoList } from '../redux/cryptosreducer/cryptosreducer';
+import CryptoItemList from './cryptoList';
+
+const MyCryptoCurrencies = () => {
+  const cryptos = useSelector((state) => state.cryptolist);
+  /* console.log(cryptos); */
```
Comment (victorokoroji) [inline src/components/cryptohome.js]: - [ ] Kindly remove commented code from your codebase so as to keep your codebase clean and readable 😄

File: src/components/cryptoList.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+/* eslint-disable max-len */
+import React from 'react';
+import PropTypes, { Object } from 'prop-types';
+import { Link } from 'react-router-dom';
+import CryptoItem from './cryptoItemStructure';
+import MyCoin from './coin/coin';
+
+const CryptoItemList = (props) => {
+  const { cryptos, search } = props;
+  /* console.log(cryptos); */
```
Comment (victorokoroji) [inline src/components/cryptoList.js]: - [ ] Kindly remove commented code from your codebase so as to keep your codebase clean and readable 😄

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @angeluray, 

Good job so far! 🤝 
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

- Good video presentation ✔️ 
- Pull request summary is descriptive and detailed ✔️ 
- Good commit messages ✔️ 
- Code is clean and well organized ✔️ 

## Suggested changes
_Check the comments under the review._

- [ ] Kindly ensure that your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rustam-tolipov — APPROVED** [review]
## Hello @angeluray👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

No Linter errors 💯

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**